spinalcordtoolbox/scripts/sct_deepseg.py

```diff
@@ -232,6 +233,16 @@ def main(argv: Sequence[str]):
                 if contrast != "stir":
                     parser.error("Task 'seg_sc_ms_lesion_stir_psir' requires the flag `-c` to be either psir or stir.")
 
+        if 'seg_sc_epi' in arguments.task[0]:
+            for image in arguments.i:
+                image_load = nb.load(image)
```

Comment:
I don't think it's necessary to use `nibabel` to load the image, given that `sct_deepseg` already imports SCT's own image-loading utility (the `Image` class).

Reply:
You're right, fixed this!